pkg/api/v1/atlascluster_types.go

```diff
@@ -125,6 +125,10 @@ type AtlasClusterSpec struct {
 	// +optional
 	AdvancedClusterSpec *AdvancedClusterSpec `json:"advancedClusterSpec,omitempty"`
 
+	// Backup schedule for the AtlasCluster
+	// +optional
+	BackupSchedule ResourceRefNamespaced `json:"backupRef"`
```

Comment:
should we be a bit more explicit with the name? Maybe "backupScheduleRef"? What do you think?

Reply:
Actually, yes, let me rename it to `backupScheduleRef`